corehq/tabs/uitab.py

```diff
@@ -149,7 +149,11 @@ def url(self):
 
     @property
     def url_prefixes(self):
-        return [url_prefix_format.format(domain=self.domain)
+        # Use self._request.domain instead of self.domain to generate url-prefixes
+        # because the latter might have a normalized domain name which might not match the
+        # domain name mentioned in the URL. for example domain-name 'hq_test' is normalized to
+        # 'ht-test'
```

Comment:
I wonder if ```self.domain``` should be set to ```self._request.domain```?  If this is a problem here I'd be concerned that it might be a problem elsewhere too.